bokehjs/src/coffee/tool/inspectors/hover_tool.coffee

```diff
@@ -141,8 +141,11 @@ class HoverToolView extends InspectTool.View
         [rx, ry] = [vx, vy]
 
       vars = {index: i, x: x, y: y, vx: vx, vy: vy, sx: sx, sy: sy, data_x: data_x, data_y: data_y}
+
       tooltip.add(rx, ry, @_render_tooltips(ds, i, vars))
 
+    @mget('callback')?.execute({index: i, x: x, y: y, sx: sx, sy: sy})
```

Comment:
I'm not sure whether we should trigger the callback here or somewhere else in order to better "manage" it. Reason is that we call the callback here for every renderer that has a "selection". With this approach the user will have to write some JS logic it he/she needs to manage a "unique" hover tool callback and not a "callback for every renderer"..

If we are sticking with this approach we should pass the renderer as well (and maybe the geometry?) otherwise it'll be tricky to understand which renderer the callback is being called for.